OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -176,6 +189,18 @@ public void Repulse()
 			SetPosition(self, CenterPosition + FlyStep(speed, repulsionForce.Yaw.Facing));
 		}
 
+		public void DisableRepulsing()
```

Comment:
The other alternative is creating a manager that control the queue and the positions of the helicopters to resupply. I could not find any other solution for this. The repulsion from others helicopters sometime knocks the helicopter that was descending away from the supply pad.
I see as two options. Create this controller that handle all positioning of the helicopters or disable the repulsion (maybe there should be a better way of disabling this).
But i really all hear to what do you thing is better. In the case of the controller, think I need a example class that I can base my work or just a north on how this should be done.